#!/usr/bin/env python3

cclauss opened the pull request.

Commit:
#!/usr/bin/env python3

neozhaoliang commented:
@cclauss : All the scripts in this repo can be run both with python 2 or 3. Will people without python 2 installed can also run it if this line is added?

Commit:
Default to Python without a version number

I will change to default to Python without a version number.

With the proposed shebang line, all of the following should work on the command line:
* python mandelbrot.py
* python2 mandelbrot.py
* python3 mandelbrot.py

The command that does not work without this shebang line is:
* ./mandelbrot.py  # assumes that `chmod +x mandelbrot.py` has been run
